cluster/ci/config/prow/plugins.yaml

```diff
@@ -1127,7 +1127,7 @@ config_updater:
       name: ci-operator-master-configs
       additional_namespaces:
       - ci-stg
-    ci-operator/config/**/*master__variant.yaml:
+    ci-operator/config/**/*master_*.yaml:
```

Comment:
> This will need to land in a PR before yours

Created the pr https://github.com/openshift/release/pull/4898 with little modification in file name format

> I would suggest literally using one file that has ```__variant```

```__variant``` is understandable but it doesn't look precise. i am expecting generator should add more information to ```trigger:```. For example
```
trigger: (?m)^/test( | .* )v4.1-integration,?($|\s.*)
```
As per your suggestion i just updated the variant config file format through a separate pr https://github.com/openshift/release/pull/4898. 

Please have a look.

Reply:
> > This will need to land in a PR before yours
> 
> Created the pr #4898 with little modification in name format
> 
> > I would suggest literally using one file that has `__variant`
> 
> `__variant` is understandable but it doesn't look precise. i am expecting generator should add more information to `trigger:`. For example
> 
> ```
> trigger: (?m)^/test( | .* )integration-v4.1,?($|\s.*)
> ```
> 
> instead of `trigger: (?m)^/test( | .* )integration-variant,?($|\s.*)`
> 
> As per your suggestion i just updated the variant config file format through a separate pr #4898.
> 
> Please have a look.

@stevekuznetsov I don't think it's valid context in this context. Please ignore my above comment.